docs/api/endpoint.md

```diff
@@ -5,7 +5,18 @@ title: The GraphQL Endpoint
 
 ## Endpoint
 
-API is available under `/graphql` endpoint. Requests must be sent using HTTP `POST` method and `application/json` content type.
+Our API is available under `/graphql` endpoint. The endpoint remains constant regardless of the operation you perform.
+
+Requests must be sent using HTTP `POST` method and `application/json` content type.
+
+To view and try our API out, visit the [Playground](https://demo.getsaleor.com/graphql/).
+
+
+
+
+
+
+
+
```

Comment:
We need to remove all those blank lines and only keep one (again, the `end-of-file-fixer` hook would fix those).